onward/app/feed/MostPopularAgent.scala

```diff
@@ -58,9 +58,11 @@ object GeoMostPopularAgent extends Logging with ExecutionContexts {
       } yield {
         getResponse(ContentApiClient.item(urlToContentPath(url), Edition.defaultEdition))
           .map(_.content.map(RelatedContentItem(_)))
-          .fallbackTo{
-            log.error(s"Error requesting $url")
-            Future.successful(None)}
+          .recover {
+            case e: Throwable =>
```

Comment:
Generally would prefer matching on `scala.util.control.NonFatal` rather than all throwables. `NonFatal` still allows things like Out of Memory, and other fatal errors to fail.